.golangci.yml

```diff
@@ -8,18 +8,25 @@ linters:
     - bodyclose
     - depguard
     - gofmt
+    - goprintffuncname
     - interfacer
+    - rowserrcheck
     - typecheck
     # - deadcode
```

Comment:
Can you change it to a blacklist instead of a whitelist? This way, we can keep track easier of which checks we don't yet enable while making updates easier. Could be a separate PR if you prefer.

Reply:
Sure, I added the comments to the `disable` section in the file.